### ISSUE TITLE
aria-haspopup incorrectly used for ARIA 1.2 Editable Combobox With List Autocomplete Example

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/aria-practices#1323]
Could someone please explain why aria-haspop is set to true when it is followed by a role listbox in the example on the page 'Editable Combobox With List Autocomplete Example'
[https://www.w3.org/TR/wai-aria-practices-1.2/examples/combobox/combobox-autocomplete-list.html](url)

`<input id="cb1-input" class="cb_edit" type="text" role="combobox" aria-autocomplete="list" aria-expanded="true" aria-controls="cb1-listbox" aria-haspopup="true" aria-activedescendant="">`

`<ul id="cb1-listbox" role="listbox" aria-label="States" tabindex="-1" class="" style="display: block;">`

> A popup element usually appears as a block of content that is on top of other content. Authors MUST ensure that the role of the element that serves as the container for the popup content is menu, listbox, tree, grid, or dialog, and that the value of aria-haspopup matches the role of the popup container.

**Comment by carmacleod** [comment]
Hi, @LaurenceRLewis!
> explain why aria-haspop is set to true when it is followed by a role listbox

Because the fix hasn't been merged yet. :)
(The fix deletes aria-haspopup from combobox examples that have a listbox popup).

The PR we are waiting on merging is https://github.com/w3c/aria-practices/pull/1276.
We wanted to test some old issues first (to make sure the PR resolves them) before merging.

@mcking65 I tested the [5 old combobox issues listed on today's agenda](https://github.com/w3c/aria-practices/wiki/February-4%2C-2020-Meeting#aria-12-combobox), and PR https://github.com/w3c/aria-practices/pull/1276 resolves them all (with the possible exception of a bit of editorial work in https://github.com/w3c/aria-practices/issues/785 that can be looked at separately). I think it can be merged now.

I'm closing this issue as a duplicate of https://github.com/w3c/aria-practices/issues/1261#issuecomment-553824167.
